configure.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+echo "Running configuration script"
+export DEBIAN_FRONTEND=noninteractive
+
+echo "US/Central" | sudo tee /etc/timezone
+dpkg-reconfigure --frontend noninteractive tzdata
+
+echo "Install essential software pacakges"
+apt-get -qq update
+apt-get -qq install -y wget htop wget build-essential zip software-properties-common gnupg awscli
```

Comment:
Cool.  Are these deps also documented in the main `torque` and/or `torque-sites` repositories, if that would be appropriate to do?

Reply:
These are mostly used for the installation (eget) / box administration (e.g. htop) -- not by torque or torque-sites directly.

... and then you have things like awscli which I missed when trying to remove from the original permanent/devenv repo that this was borrowed from!

I'll revise to make sure it's lean (and if any *are* needed for the install!)

Reply:
I've... basically removed everything (when I had these, I hadn't been running many ansible scripts; just about everything is actually installed by those, and even apache2 would be but I want to install it here so I can mod the permissions of /var/www)